packages/release/Cargo.toml

```
@@ -16,6 +16,7 @@ path = "pkg.rs"
 [dependencies]
 acpid = { path = "../acpid" }
 ca-certificates = { path = "../ca-certificates" }
+binutils = { path = "../binutils" }
```

super nit: should keep this sorted.
```suggestion
binutils = { path = "../binutils" }
ca-certificates = { path = "../ca-certificates" }
```